hw9/hw9_t01.py

```diff
@@ -17,32 +17,51 @@
 from typing import Iterator, List, Union
 
 
-def open_file_return_list(path: Union[Path, str]) -> list:
-    with open(path, "r") as file:
-        return list(map(int, file.read().split()))
+def open_file_return_iterator(path: Union[Path, str]) -> Iterator:
+    with open(path) as file:
```

Comment:
Is it a good practice to skip default parameters?
`with open(path, "r", encoding="utf-8") as file:`

Reply:
Yes, It is good practice to skip default arguments.